### ISSUE TITLE
Multiple IRC servers/networks

### ISSUE THREAD

**Issue by k0nsl** [g0v/Logbot#37]
What do you all think about adding support for multiple IRC servers/networks? Would it be possible without too much effort/hassle?

**Comment by godfat** [comment]
Do you mean to connect to multiple IRC servers?

**Comment by k0nsl** [comment]
@godfat: Yes, and then log those networks as well. That would require some major refactoring, though, now that I think about it more clearly.

**Comment by godfat** [comment]
@k0nsl I just took some look, and I think we just need to have some prefix for the key and think about how we should do with the URL, and backward compatibility. Logbot isn't too complex yet, so it should be pretty doable.

However, what do you think about just run another Logbot instance? If that's also easy/simple enough, we probably don't need to bother with this.

**Comment by k0nsl** [comment]
It would be awesome to be able to add multiple IRC networks in the same Logbot instance just to keep things more 'centralized' which also means there is no need for multiple domains/subdomains.

**Comment by godfat** [comment]
With a reverse proxy like Nginx, we don't really need multiple domains/subdomains.

A single Redis instance could also provide multiple databases.

The advantage of running the same Logbot instance would be left to easier to manage Logbot processes, and more flexibility (considering that there might have environments which cannot allow multiple processes, for example)

I could see it's beneficial, certainly, but I don't see a huge advantage here.

On the other hand, of course pull requests are always welcome, considering that effort is the only concern. I would be happy to review the codes, if you're going to give it a try :)
